projects/opendr_ws_2/src/opendr_planning/opendr_planning/end_to_end_planning_robot_driver.py

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env python
+# Copyright 2020-2022 OpenDR European Project
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import rclpy
+
+
+class EndToEndPlanningUAVRobotDriver:
+    def init(self, webots_node, properties):
+        rclpy.init(args=None)
+        self.__node = rclpy.create_node('end_to_end_planning_uav_robot_driver')
```

Comment:
This class seems to be used as `plugin type` [here](https://github.com/opendr-eu/opendr/blob/778f36698447911772cdb85dcd04753aee6216fc/projects/opendr_ws_2/src/opendr_planning/resource/uav_robot.urdf#L32). I am not exactly familiar to how this is used, but both arguments seem unused and init is missing the underscores. If they are supposed to be this way, feel free to ignore this and resolve the comment.

Reply:
no, it has to be init without underscores otherwise the plugin is not built correctly

Reply:
Then resolving this comment without action.